lib/http/node.js

```diff
@@ -48,7 +48,7 @@ NodeHttpClient.prototype.getAgentAndProxyOptions = function (url, proxy, mock)
 {
   const isHttps = Url.parse(url).protocol === 'https:';
   let agentClass;
-  let agentOptions = { keepAlive: true };
+  let agentOptions = { keepAlive: true, keepAliveMsecs: 60000};
```

Comment:
Question: Why I propose so long keep alive?

Reply:
it's removed from this PR